Source/WebKit/UIProcess/API/wpe/WebKitSettings.h

```diff
@@ -547,6 +547,13 @@ WEBKIT_API void
 webkit_settings_set_webrtc_udp_ports_range                     (WebKitSettings *settings,
                                                                 const gchar    *udp_port_range);
 
+WEBKIT_API gboolean
+webkit_settings_get_screen_supports_hdr                        (WebKitSettings* settings);
+
+WEBKIT_API void
+webkit_settings_set_screen_supports_hdr                        (WebKitSettings* settings,
+                                                                const gboolean screenSupportsHDR);
```

Comment:
My personal preference is to avoid `const` for simple data type arguments

```suggestion
                                                                gboolean screenSupportsHDR);
```

Reply:
and it will be consistent with other similar functions. I will correct it. Thanks